awx/main/migrations/0061_v350_add_openstack_cedential_verify_ssl.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+# AWX
+from awx.main.migrations import _credentialtypes as credentialtypes
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0060_v350_update_schedule_uniqueness_constraint'),
+    ]
+
+    operations = [
+        migrations.RunPython(credentialtypes.add_openstack_verify_field),
+        migrations.RunPython(credentialtypes.migrate_openstack_credentials),
```

Comment:
I don't think this step is necessary because the default is True, so there's no need to add the field to all the existing credentials.

Reply:
@AlanCoding Thank you for your advice. I have removed the migration logic for the existing openstack credentials.
Please let me know if it does not sound correct.